mypy/nodes.py

```diff
@@ -136,9 +136,13 @@ def get_column(self) -> int:
     'builtins.frozenset': 'typing.FrozenSet',
 }  # type: Final
 
-nongen_builtins = {'builtins.tuple': 'typing.Tuple',
+_nongen_builtins = {'builtins.tuple': 'typing.Tuple',
                    'builtins.enumerate': ''}  # type: Final
-nongen_builtins.update((name, alias) for alias, name in type_aliases.items())
+_nongen_builtins.update((name, alias) for alias, name in type_aliases.items())
+
+def get_nongen_builtins(python_version):
```

Comment:
```suggestion
def get_nongen_builtins(python_version):  # type: (Tuple[int, int]) -> dict
```

Reply:
Actually, this should use inline type annotations (not type comments), and a more precise type than `dict` (I haven't looked what it should be).

Reply:
Perhaps it's a good idea to either type the function as returning `Mapping[str, str]`, or make it return either copies of the original dict or something actually immutable like `types.MappingProxyType`.